backend/jottacloud/jottacloud.go

```diff
@@ -92,6 +92,33 @@ func init() {
 		Description: "Jottacloud",
 		NewFs:       NewFs,
 		Config:      Config,
+		MetadataInfo: &fs.MetadataInfo{
+			Help: `Jottacloud has limited support for metadata, currently an extended set of timestamps.`,
+			System: map[string]fs.MetadataHelp{
+				"btime": {
+					Help:    "Time of file birth (creation), read from rclone metadata",
+					Type:    "RFC 3339",
+					Example: "2006-01-02T15:04:05.999999999Z07:00",
+				},
+				"mtime": {
+					Help:    "Time of last modification, read from rclone metadata",
+					Type:    "RFC 3339",
+					Example: "2006-01-02T15:04:05.999999999Z07:00",
+				},
+				"utime": {
```

Comment:
We should probably put `utime` in the standard table?

https://rclone.org/docs/#standard-system-metadata

Reply:
Sure.
Jottacloud uses term "updated", which I think is confusing to understand what represents compared to the last modified timestamp, so I suggest we make the "u" stand for "uploaded" and document it as "Time of file upload"? Transfer to a local system which happens to support it would for a user be more of a "download", but not a huge problem I think 😅 , although I'm open for other suggestions.

Reply:
Yes `utime` == uploaded time sounds perfect to me.

> Transfer to a local system which happens to support it would for a user be more of a "download", but not a huge problem I think 😅 , although I'm open for other suggestions.

He he! Happy with `utime`!